### ISSUE TITLE
Add alt attribute to the images and assets used in Concepts page

### ISSUE THREAD

**Issue by EshaanAgg** [novuhq/docs#194]
Many menu screenshots in the [Concepts page in the guide](https://docs.novu.co/getting-started/concepts) have no alt text associated with them. This makes the site less accessible to screen readers. This should be addressed.

**Comment by EshaanAgg** [comment]
I would like to work on this issue.

**Comment by unicodeveloper** [comment]
Please pick from the existing issues. We don't want to attend to this specific type of issue for now during #hacktoberfest.

**Comment by EshaanAgg** [comment]
@unicodeveloper I totally respect your decision not to attend this issue during hacktoberfest, but I still would urge you to keep this issue open (maybe with a `non-hacktoberfest` tag or something?) because I feel this is a real lacking from the accessibility point of view!

But it's totally your call. Thanks for your time.
